AGGREGATE.md

```diff
@@ -254,7 +254,9 @@ utilize techniques like retries to minimize data loss.
 Note: if [debugging](https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#optional-extended-debugging-reports)
 is enabled, additional debug fields will be present in aggregatable reports,
 including the cleartext payloads, to allow downstream systems to verify that
-reports are constructed correctly.
+reports are constructed correctly. If debugging is enabled, the `shared_info` 
+will include the flag `debug_mode=true` to let the aggregation service know
```

Comment:
It should be in JSON so "debug_mode: 1" or something like that

Reply:
still should be `debug_mode: 1` to be valid.

Reply:
should be fixed now?

Reply:
oops, now new and improved, with a colon